packages/react-components/src/Tags.tsx

```diff
@@ -18,10 +18,11 @@ interface Props {
   onToggleIsEditing?: () => void;
   onSave?: () => void;
   value: string[];
+  withEditButton?: boolean;
```

Comment:
Is there any place we still use the edit button? Can't find it.

Even if we do, I feel like we have to many props. Should be enough to have `isEditing` (for showing inputs) and `isEditable` (for showing button).

Reply:
It is also used in Contracts and there we don't provide button for editing like in sidebar. 
![image](https://user-images.githubusercontent.com/30404388/132247187-519d116a-70a1-4412-841b-ac3ed7a5721c.png)